server/channels/store/searchlayer/channel_layer.go

```diff
@@ -100,11 +100,8 @@ func (c *SearchChannelStore) UpdateMember(rctx request.CTX, cm *model.ChannelMem
 	return member, err
 }
 
-func (c *SearchChannelStore) SaveMember(cm *model.ChannelMember) (*model.ChannelMember, error) {
-	// TODO: Use the actuall request context from the App layer
```

Comment:
removed these comments & assignment as ctx loaded from App.

Reply:
Yes, thank you :+1: